ska_conda/ska_builder.py

```diff
@@ -59,30 +81,20 @@ def _build_package(self, name):
         print("Building package %s." % name)
         pkg_path = os.path.join(pkg_defs_path, name)
         cmd_list = ["conda", "build", pkg_path, "--croot",
-                    self.ska_build_dir, "--no-test",
-                    "--no-anaconda-upload"]
+                    self.build_dir, "--no-test",
```

Comment:
It may take a little longer to build, but I'd prefer to fix the packages as needed so we can actually run the tests if provided.

Reply:
Can you elaborate, I don't understand this comment.

Reply:
I'd prefer to run the tests and fix any tests that need fixing, instead of skipping the tests with "--no-test".

Reply:
Oh, and testing or not, I think we should check the status of the conda build command and do something with it (stop or save to report at the end).

Reply:
As you know I'm not a big fan of build time testing, but we can discuss later when we have the core requirements all in place.

:+1: on checking the build command status.  Just call with `check=True, timeout=120` so it raises an exception if anything went wrong or it takes too long.

Reply:
Maybe a test option for the build?  Mentioning because we haven't defined a process to make new recipes.  If we just plop them in the repo, the easiest way to determine they are complete and correct is to run the tests.  For example, by running the build tests, I just discovered that the maude recipe needs to have the requests package added as a runtime dependency/requirement.  Of course, I could run the build and build tests outside the ska_builder process, but I think that might be over-complicated.

Reply:
Also, that timeout won't work for some builds, so I'm not sure if it would be better to just not define one for now.